tmt/steps/provision/vagrant.py

```diff
@@ -150,35 +147,48 @@ def prepare(self, how, what):
                 self.kv('run', 'never'),
                 self.kv(method, what))
 
-            # maybe?
-
         return self.run_vagrant(cmd, f'--{cmd}-with', name)
 
 
     ## Additional API ##
     def init(self):
-        """ Initialize Vagrantfile """
+        """ Initialize ProvisionVagrant / run following:
+            1] check that Vagrant works
+            2] check for already-present or user-specified Vagrantfile
+            3] check input values and set defaults
+            4] create and populates Vagrantfile with
+                - provider-specific entries
+                - default config entries
+        """
         self.info('Provision dir', self.provision_dir)
```

Comment:
I would make this just a debug message and start with small letters, so the output is consistent

Reply:
Sure. (Generally, I find this very important and informative, as I CD to the provision dir very often... but that may not be a general use.)

Reply:
yeah :) it is not for general consumption

Reply:
Yes, should be moved to debug. Info output should still be relatively concise.